GliaWidgets/Component/Connect/Operator/ConnectOperatorView.swift

```diff
@@ -88,6 +98,24 @@ class ConnectOperatorView: UIView {
         animationView = nil
     }
 
+    func showOnHoldView() {
+        guard let style = style.onHoldOverlay else { return }
+
+        let onHoldView = OnHoldOverlayView(
+            style: style,
+            imageSize: .init(width: 40, height: 40)
```

Comment:
Let's move image size to dedicated constant.

Reply:
the size varies depending whether it is in bubble or in call view though